jekyll/_cci2/local-cli.md

```diff
@@ -7,170 +7,311 @@ categories: [troubleshooting]
 
-The CLI upgrade provides additional functionality through a totally new CLI developed as a [CircleCI-Public open source project](https://github.com/CircleCI-Public/circleci-cli). If you have the old CLI installed, run the following commands to update and switch to the new CLI:
+There are multiple installation options for the CLI.
 
-```
-circleci update
-circleci switch
-```
+### Quick Installation
 
-This command may prompt you for `sudo` if your user doesn't have write permissions to the install directory, `/usr/local/bin`.
+**Note**: If you have already installed the CLI prior to October 2018 you may need to do an extra one-time step to switch to the new CLI. See [upgrading instructions below](#updating-the-legacy-cli).
 
-### Installing the CircleCI CLI on macOS or Linux
-{:.no_toc}
+For the majority of installations, the following commands will get you up and running with the CircleCI CLI:
 
-If you haven't already installed `circleci` on your machine, run the following command to install the CircleCI CLI:
+**Mac and Linux:**
 
-```
+```sh
 curl -fLSs https://circle.ci/cli | bash
 ```
 
-By default, the `circleci` CLI tool will be installed to the `/usr/local/bin` directory. If you do not have write permissions to `/usr/local/bin`, you may need to run the above command with `sudo`. Alternatively, you can install to an alternate location by defining the `DESTDIR` environment variable when invoking `bash`:
+By default, the CircleCI CLI tool will be installed to the `/usr/local/bin` directory. If you do not have write permissions to `/usr/local/bin`, you may need to run the above command with `sudo`. Alternatively, you can install to an alternate location by defining the `DESTDIR` environment variable when invoking bash:
 
-```
+```sh
 curl -fLSs https://circle.ci/cli | DESTDIR=/opt/bin bash
 ```
 
-### [Alternative] Install with snap 
+### Alternative Installation Methods
+
+There are several alternative installation methods for the CircleCI CLI. Read on if you need to customize your installation, or are running into issues with the quick installation above.
+
+#### Install with Snap
 {:.no_toc}
 
-The following commands will install the CircleCI CLI, Docker, and the security and auto-update features that come along with [Snap packages](https://snapcraft.io/).
+The following commands will install the CircleCI CLI, Docker, and the security and auto-update features that come along with Snap packages.
 
-```
+```sh
 sudo snap install docker circleci
 sudo snap connect circleci:docker docker
 ```
 
-**Note:** With snap packages, the `docker` command will use the Docker snap, not any version of Docker you may have previously installed. For security purposes, snap packages can only read/write files from within `$HOME`.
+**Note**: With snap packages, the docker command will use the Docker snap, not any version of Docker you may have previously installed. For security purposes, snap packages can only read/write files from within $HOME.
```

Comment:
I think this formatting was correct before, colon goes inside the bold and we should use backticks for commands and $HOME